### PR TITLE
Updated the Eclipse Usage information

### PR DESCRIPTION
**Pull request by keinhaar**
- Fixed Plugin Name
- Linked to Github Plugin Site for Installation Instructions
- Correct names for Menu Entries
- removed obsolete screenshots
- removed section for app engine

### CODE REVIEW

File: src/main/markdown/usingeclipse.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Using Eclipse
 
 GWT provides a set of tools that can simply be used with a
 text editor, the command line, and a browser. However, you may also use GWT with your
-favorite IDE. Google provides a plugin for Eclipse that makes development with
+favorite IDE. The GWT Plugins Project provides a plugin for Eclipse that makes development with
```
[the remark this answers is not in the document]
Comment (keinhaar) [reply src/main/markdown/usingeclipse.md]: Added the link

File: src/main/markdown/usingeclipse.md [hunk 2]
```diff
@@ -19,31 +19,23 @@ If you do not already have Eclipse, you may download it from the [Eclipse Websit
 
 ## Install the Plugin<a id="installing"></a>
 
-<img src="https://developers.google.com/eclipse/images/google-plugin.png" style="float: left; width: 40px;
-margin-right: 15px;"/>
+Install the GWT Plugin for Eclipse following [these instructions](https://github.com/gwt-plugins/gwt-eclipse-plugin)
 
-Install the Google Plugin for Eclipse following [these instructions](https://developers.google.com/eclipse/docs/download)
+The best way to install is by using the Eclipse Marketplace.
```
[the remark this answers is not in the document]
Comment (keinhaar) [reply src/main/markdown/usingeclipse.md]: Added the link

File: src/main/markdown/usingeclipse.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhillbounce) [reply src/main/markdown/usingeclipse.md]: Question.  It seems the best practice (based on Gitter Discussions) is for any NEW GWT development to use Maven.  Should we add a statement like....
"Using the GWT Eclipse Plugin" is no longer the preferred development method and it is suggested NEW projects be setup using Maven (with a link to setting up a Maven GWT project in Eclipse).  Thoughts?

[the remark this answers is not in the document]
Comment (niloc132) [reply src/main/markdown/usingeclipse.md]: I was thinking I'd propose a new section here, before the sample gets started, to redirect the user to set up gradle or maven if  they know those tools, then return here for import, and if they only want it to work on their machine (or carefully curate eclipse config checkins, and not plan on supporting CI directly, etc), then continue for how to set up a sample.

For maven there are at least only a few specific starting points, for gradle there are myriad gwt plugins and forks, and I don't know how we would pick the "right" one...

[the remark this answers is not in the document]
Comment (keinhaar) [reply src/main/markdown/usingeclipse.md]: I think adding some maven and/or gradle related infos would be nice, but this is the "Using Eclipse" page. So wouldn't it be better to create separate pages for those topics?
Also these could be added as separate pull requests.  
This one is mainly for updating outdated and misleading information.

**Comment by keinhaar** [comment]
Removed beta version infos and added marketplace install button.
Also added a link to the github release page.

**Comment by protoism** [comment]
Hi all, it is a pity that this PR is stopped.
Considering that the [gwtproject.org](https://www.gwtproject.org/usingeclipse.html) is badly outdated, we might close this one and open a follow-up ticket

**Pull request merged**